src/main/java/io/github/syst3ms/skriptparser/parsing/SyntaxParser.java

```diff
@@ -64,6 +64,12 @@
     // Gradle requires the cast, but IntelliJ considers it redundant
     public static final PatternType<Object> OBJECTS_PATTERN_TYPE = new PatternType<>((Type<Object>) TypeManager.getByClass(Object.class).orElseThrow(AssertionError::new), false);
 
+    public static final ExpressionInfo<ExprBooleanOperators, Boolean> EXPRESSION_BOOLEANOPERATORS
```

Comment:
Might need some testing to check that this isn't being initialized until after registration.

Reply:
I did some testing and no errors appear, which is normal considering we don't use the class until after registration. Unless we change things drastically in the future, this should be fine.